src/storage/src/table/state_table.rs

```diff
@@ -45,13 +45,16 @@ pub struct StateTable<S: StateStore> {
 
     /// Relation layer
     cell_based_table: CellBasedTable<S>,
+
+    _pk_index: Vec<usize>,
 }
 impl<S: StateStore> StateTable<S> {
     pub fn new(
         keyspace: Keyspace<S>,
         column_descs: Vec<ColumnDesc>,
         order_types: Vec<OrderType>,
         dist_key_indices: Option<Vec<usize>>,
+        _pk_index: Vec<usize>,
```

Comment:
Why started with an underscore (`_`) ?

Reply:
By the way, recommend `pk_indices`

Reply:
By the way *2, `OrderPair` looks better than `order_types & pk_index` ?

Reply:
It hasn't been used yet, and will be used in top_n_state.